assets/css/custom.css

```diff
@@ -147,3 +147,25 @@ color: #e96958;
 }
 
 .navigation-bar > li.open > ul.dropdown-menu a { color: #e96958; width: 100% }
+
+#ribbon {
+  font-size: 12px;
+  z-index: 100000;
+  position: absolute;
+  top: 0;
+  background-color: #e96958;
+  color: white;
+  padding: 3px 10px 5px 10px;
+  text-align: center;
+  top: 25px;
+  left: -70px;
+  -webkit-transform: rotate(-45deg);
+  -moz-transform: rotate(-45deg);
+  -ms-transform: rotate(-45deg);
+  -o-transform: rotate(-45deg);
+  transform: rotate(-45deg);
```

Comment:
`transform` without prefixes is supported in IE10 (2012) and forward — I think you can drop the additional prefixed rules, if you’d like!

Reply:
Good to know, thanks!